control-plane/api-gateway/cache/gateway.go

```diff
@@ -14,16 +14,13 @@ import (
 	"github.com/hashicorp/consul-k8s/control-plane/consul"
 	"github.com/hashicorp/consul/api"
 	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/event"
 )
 
 type GatewayCache struct {
 	config    Config
 	serverMgr consul.ServerConnectionManager
 	logger    logr.Logger
 
-	events chan event.GenericEvent
```

Comment:
Confirmed that this channel isn't being used anywhere. It looks like it was a copy-paste leftover in a previous PR